src/bp/nlu-core/package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
**pointing the attention of reviewers on this file**

Reply:
This file is not necessary, if you move your typings in `common` you can remove it. 

Don't hesitate to speak with me when you have these kind of changes to make, I spent a lot of time to make sure everything works fine between components, and to ensure they can be extended easily for most use cases.

Reply:
mmmhhh pros are:

1. makes it obvious that this is now a standalone package.
2. makes it obvious what is "package public" from what is "package private" -> index.ts and nlu-core.d.ts

I requested your review on this pull request so I could have your feedback. I'm also spending a lot of time to make sure everything works. Sorry to teach you, but a codebase, it moves over time.

Reply:
Actually, it isn't a standalone package and it's just misleading. It is still built by the core, and part of the core. I see no reason to have this package for now. We will split the codebase, but we have to think about how it will be done.